splunk/packages/edit-table/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@marcusschiesser/edit-table",
-  "version": "1.5.2",
+  "version": "1.5.5",
```

Comment:
@thucpn according to semantic versioning, this is not a patch (1.5.2 -> 1.5.5), but at least a new minor version